### DISCUSSION TITLE
What is the highest point in the United States?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#469] [Galaxy Trivia] [answered]
**Bonus:** In which U.S. state is it located?

**Comment by ralphcajipe** [comment, marked as answer]
Denali is the highest point in the United States. It is located in the state of Alaska and is 20,310 feet tall. It was formerly known as Mount McKinley but was renamed Denali in 2015.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/ralphcajipe?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
